FreeRTOS/FreeRTOS-Kernel
# Rename ARM_CA53_64_BIT/_SRE to Arm_AARCH64/_SRE

Description
-----------

The Cortex-A53 ports are generic and can be used as a starting point for other Armv8-A application processors. Therefore, rename `ARM_CA53_64_BIT` to `Arm_AARCH64` and `ARM_CA53_64_BIT_SRE` to `Arm_AARCH64_SRE`.

With this renaming, existing projects that use old port, should migrate to renamed port as follows:

* `ARM_CA53_64_BIT` -> `Arm_AARCH64`
* `ARM_CA53_64_BIT_SRE` -> `Arm_AARCH64_SRE`

Test Steps
-----------

There are no functional changes.

Checklist:
----------



Related Issue
-----------

https://github.com/FreeRTOS/FreeRTOS-Kernel/issues/565


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
